### PR TITLE
posting 2.3.0 (new formula)

### PR DESCRIPTION
**Pull request by daeho-ro**
replace 
- https://github.com/Homebrew/homebrew-core/pull/177813

There is a non-PyPI resource and so pypi_formula_mapping may not work correctly.
---

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/p/posting.rb [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+  resource "textual-autocomplete" do
+    url "https://files.pythonhosted.org/packages/bd/d3/7837e2ee1807c72e2a8a185c6e5e729dbe68161d8476055d989f3a2db348/textual_autocomplete-3.0.0a13.tar.gz"
+    sha256 "db5a4ae956dd7d6dece53f7f695e97f2ab75819dd96b8a1c064ec5966b3ab113"
+  end
+
+  resource "tree-sitter" do
+    url "https://files.pythonhosted.org/packages/39/9e/b7cb190aa08e4ea387f2b1531da03efb4b8b033426753c0b97e3698645f6/tree-sitter-0.21.3.tar.gz"
+    sha256 "b5de3028921522365aa864d95b3c41926e0ba6a85ee5bd000e10dc49b0766988"
+  end
```
Comment (SMillerDev) [inline Formula/p/posting.rb:145]: Isn't this a formula?

File: Formula/p/posting.rb [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+  resource "anyio" do
+    url "https://files.pythonhosted.org/packages/9f/09/45b9b7a6d4e45c6bcb5bf61d19e3ab87df68e0601fa8c5293de3542546cc/anyio-4.6.2.post1.tar.gz"
+    sha256 "4c8bc31ccdb51c7f7bd251f51c609e038d63e34219b44aa86e47576389880b4c"
+  end
+
+  resource "brotli" do
+    url "https://files.pythonhosted.org/packages/2f/c2/f9e977608bdf958650638c3f1e28f85a1b075f075ebbe77db8555463787b/Brotli-1.1.0.tar.gz"
+    sha256 "81de08ac11bcb85841e440c13611c00b67d3bf82698314928d0b676362546724"
+  end
```
Comment (SMillerDev) [inline Formula/p/posting.rb:30]: Isn't this a formula?

File: Formula/p/posting.rb (hunk 1 above)
Comment (daeho-ro) [reply Formula/p/posting.rb:145]: You are right! Should replace 😃

File: Formula/p/posting.rb [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    url "https://files.pythonhosted.org/packages/bd/d3/7837e2ee1807c72e2a8a185c6e5e729dbe68161d8476055d989f3a2db348/textual_autocomplete-3.0.0a13.tar.gz"
+    sha256 "db5a4ae956dd7d6dece53f7f695e97f2ab75819dd96b8a1c064ec5966b3ab113"
+  end
+
+  # sdist issue report, https://github.com/grantjenks/py-tree-sitter-languages/issues/63
+  # use fork until resolved
+  resource "tree-sitter-languages" do
+    url "https://github.com/Textualize/py-tree-sitter-languages/archive/refs/tags/v1.11.0b0.tar.gz"
+    sha256 "17b13f25f479e1ca39cddab1ec59541236ae25449ba136e70e842c3c8340c507"
+  end
```
Comment (chenrui333) [inline Formula/p/posting.rb:144]: at this point, py-tree-sitter-languages is unmaintained.

this issue might be a better ref, https://github.com/Aider-AI/grep-ast/issues/7 and still have active discussions there.

**Review by chenrui333 — APPROVED** [review]

File: Formula/p/posting.rb (hunk 3 above)
Comment (darrenburns) [reply Formula/p/posting.rb:144]: Hi, author of Posting here, FWIW I'll be moving away from `py-tree-sitter-languages` very soon (likely within the next week or so).

Comment (chenrui333) [reply Formula/p/posting.rb:144]: that is good to hear, thanks @darrenburns!

maybe once the change landed, feel free to create a PR in here.

Comment (daeho-ro) [reply Formula/p/posting.rb:144]: It does not build with python 3.13 and unmaintained repo, and so I have used forked one.
Anyway I will change the issue ref.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**